examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -836,7 +836,7 @@ def load_model_hook(models, input_dir):
             else:
                 raise ValueError(f"unexpected save model: {model.__class__}")
 
-        lora_state_dict, network_alpha = LoraLoaderMixin.lora_state_dict(input_dir)
+        lora_state_dict, network_alpha, _ = LoraLoaderMixin.lora_state_dict(input_dir)
```

Comment:
this is a breaking change - do we have to do this? Would prefer to not by default return a third variable here